Resources/Audio/Items/Recorder/attributions.yml

```diff
@@ -0,0 +1,4 @@
+- files: ["recorder_erase.ogg", "recorder_start.ogg", "recorder_stop.ogg"]
+  license: "СС-BY-NC-SA-3.0"
+  copyright: "Taken from Zvukipro.com"
```

Comment:
Can you point out where the license is listed on the site please.

Reply:
> Can you point out where the license is listed on the site please.

I didn't found exact line with license name, but it says that is free to use if you stay the site link (CC-BY-SA as i know right) - https://zvukipro.com/index.php?do=feedback

Reply:
Don't really understand much about licenses

Reply:
About old PR, i didn't saw them earlier :|. I watched them now and think, that they look much better then mine. I'll try to ask authors about them

Reply:
They don't specify a license there and they also host the obviously licensed sounds from AAA games (Zelda BotW, Prey, Bioshock Infinite). It's almost guaranteed the sound you've used is owned by someone.

The primary source for the station's sound effects is https://freesound.org/, it seems, while [this search](https://github.com/search?q=repo%3Aspace-wizards%2Fspace-station-14+.mp3+OR+.ogg+OR+.m4a+OR+.flac+path%3A**%2Fattributions.txt&type=code) will show you the attributions files and the sources for the specific sounds.